src/server/game/Entities/Creature/CreatureGroups.cpp

```diff
@@ -186,7 +186,7 @@ void CreatureGroup::MemberAttackStart(Creature* member, Unit* target)
     }
 }
 
-void CreatureGroup::FormationReset(bool dismiss)
+void CreatureGroup::FormationReset(bool dismiss, bool initiMotionMaster)
```

Comment:
is initi for initializing? if so I propose using initMotionMaster instead

Reply:
ffs typo. it was meant to be 'initMotionMaster'

Reply:
y, it was a typo my bad.